include/Match.h

```diff
@@ -290,12 +337,19 @@ class PMMatch : public Match {
         prepare();
         return pmTrueCov;
     }
+    // Return covariance matrix of PM in output units.
+    PMCovariance getFitCovariance() const;
 
     // Get predicted position (and inverse covariance) for a Detection.
     // The argument is needed only if there is full PM solution.
     Vector2 predict(const Detection *d = nullptr) const override;
+    // Get predicted position of object at epoch of each Detection.
+    std::vector<std::vector<double>> predictAtDetections() const;
```

Comment:
What is this a vector of vectors of?  Seems like one of them should just be some kind of 2-element struct (`std::pair`, if you don't have something better), but I'm not sure which.

Reply:
It's a list of RA/Dec points. Any objection to converting it to numpy in the pybind11 file? I can take advantage of the `vectorToNumpy` helper function you helped me write a long time ago. I think that's what I intended to do originally, but forgot.

Reply:
That sounds good.  I'm also fine with leaving it for this week, if that helps you finish this off this evening.